manifests/p/Plex/Plex/1.62.2/Plex.Plex.locale.en-US.yaml

```
@@ -0,0 +1,37 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-8
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Plex.Plex
+PackageVersion: 1.62.2
+PackageLocale: en-US
+Publisher: Plex, Inc.
+PublisherUrl: https://www.plex.tv/
+PublisherSupportUrl: https://support.plex.tv/
+PrivacyUrl: https://www.plex.tv/about/privacy-legal/
+Author: Plex, Inc.
+PackageName: Plex
+PackageUrl: https://www.plex.tv/
+License: Freemium, GPL-2.0
+LicenseUrl: https://www.plex.tv/about/privacy-legal/plex-terms-of-service
+Copyright: Copyright © 2021 Plex
+CopyrightUrl: https://www.plex.tv/about/privacy-legal/plex-copyright-policy/
+ShortDescription: Plex is a global streaming media service and a client–server media player platform, made by Plex, Inc.
+# Description:
+# Moniker:
```

```suggestion
Moniker: plex
```
As per the discussion on gitter